wagtail_streamfield_migration_toolkit/test/tests/test_streamchangedetector.py

```diff
@@ -0,0 +1,112 @@
+from unittest import mock
+from django.test import TestCase
+from wagtail.blocks import CharBlock, StreamBlock
+
+from wagtail_streamfield_migration_toolkit.autodetect.streamchangedetector import (
+    StreamDefChangeDetector,
+)
+from wagtail_streamfield_migration_toolkit.operations import (
+    RenameStreamChildrenOperation,
+    RemoveStreamChildrenOperation,
+)
+from ..models import (
+    BaseStreamBlock,
+    SimpleStreamBlock,
+)
+
+
+class SimpleStreamWithRemovedChild(StreamBlock):
+    # removed char 1
+    char2 = CharBlock()
+
+
+class SimpleStreamWithRenamedChild(StreamBlock):
+    renamed_char1 = CharBlock()
+    char2 = CharBlock()
+
+
+class StreamChangeDetectorTests(TestCase):
+    def assertRemoveOperationEqual(
+        self, operation_and_block_path1, operation_and_block_path2
+    ):
+        self.assertEqual(operation_and_block_path1[1], operation_and_block_path2[1])
```

Comment:
Maybe rather than indexing/slicing here, you could destructure ie `operation_1, block_path_1 = operation_and_block_path_1`. Though tbh, it may be easier just to compare operations here, and compare the block paths elsewhere